app/components/forms/wizard/basic-details-step.js

```diff
@@ -464,9 +464,9 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     updateSalesEndDate(eventStartDate) {
       eventStartDate = moment(new Date(eventStartDate));
       this.data.event.tickets.forEach(ticket => {
-        if (moment(eventStartDate).isBefore(ticket.get('salesEndsAt'))) {
+        // if (moment(eventStartDate).isBefore(ticket.get('salesEndsAt'))) {
```

Comment:
don't comment code, delete it.

Reply:
said changes done.

Reply:
need to be fixed on server too

Reply:
I need to get familiarized with the server code base then

Reply:
you need to read issue again you have made totally different change what issue is saying

Reply:
And also i guess no frontend change is needed.

Reply:
yes @maze-runnar i got it.